src/components/Table/Td.tsx

```diff
@@ -29,7 +29,7 @@ export const Td: FC<Props & ElementProps> = ({
   return (
     <StyledTd
       {...props}
-      className={`${wrapperClass} ${fixed && 'fixedElement'}`}
+      className={`${wrapperClass}${fixed ? ' fixedElement' : ''}`}
```

Comment:
imo: 余計な余白は空いちゃうけど、こっちの方が意図が伝わりやすいかも

```suggestion
      className={`${wrapperClass} ${fixed ? 'fixedElement' : ''}`}
```